Xamarin/Droid/MainActivity.cs

```diff
@@ -92,17 +112,39 @@ private void OnTouch(object sender, View.TouchEventArgs touchEventArgs)
                     needsRefresh = true;
                     break;
             }
+
             if (needsRefresh)
             {
                 _canvas.Invalidate();
             }
         }
 
+        // use the back button in preference to trying to detect shake, which is not built in
         public override void OnBackPressed()
         {
             _drawer.ErasePaths();
```

Comment:
That's somewhat unintuitive (not that shaking is), but I guess for a demo it's fine